avogadro/molequeue/molequeuewidget.cpp

```diff
@@ -237,9 +237,10 @@ void MoleQueueWidget::onSubmissionSuccess(int localId, unsigned int mqId)
   emit jobSubmitted(true);
 }
 
-void MoleQueueWidget::onSubmissionFailure(int localId, unsigned int,
-                                          const QString& error)
+void MoleQueueWidget::onSubmissionFailure(int localId, int,
```

Comment:
Why are you changing this from unsigned to signed? I think keeping the QJson along is a nice addition, but I’m not sure at using signed int throughout. What’s the motivation?

Reply:
Moreover, I’d really like to change to the newer QT5 PMF syntax, since it will prevent failures along the lines of your patches (i.e., everything just breaks at compile-time).

Thanks.